torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -28,6 +28,11 @@ def aten_getitem(self, i):
     return opset18.SequenceAt(self, i)
 
 
+@onnxscript.script(opset=TORCH_ONNX_OPSET)
```

Comment:
Could you delete this? https://github.com/pytorch/pytorch/pull/94878#event-8527713615